Dockerfile

```
@@ -2,16 +2,19 @@ ARG NODE_VERSION=24.10.0-alpine
 # dist
 FROM node:$NODE_VERSION AS dist
 
+# Install pnpm
+RUN corepack enable && corepack prepare pnpm@10.0.0 --activate
+
 WORKDIR /dep
 
 # Copy lockfile
-COPY ./yarn.lock ./.yarnrc.yml ./
-COPY .yarn .yarn
+COPY ./pnpm-lock.yaml ./pnpm-workspace.yaml ./package.json ./
```

```suggestion
COPY ./pnpm-lock.yaml ./pnpm-workspace.yaml ./
```

else pnpm fetch optimization is useless